src/services/JobRoleService.ts

```diff
@@ -14,3 +15,22 @@ export const getJobRoles = async (): Promise<JobRolesResponse[]> => {
         throw new Error('Failed to get Job Roles');
     }
 }
+
+export const getJobRoleById = async (id: string): Promise<JobRole> => {
+    try {
+        const response: AxiosResponse = await axios.get(URL + id);
+
+        if(response.data.status != "open") {
+            return null;
+        }
```

Comment:
the acceptance criteria don't ask for this so I would remove it (we'll want to be able to view closed job role information in the future

Reply:
Removed